core/java/iesi-rest-without-microservices/src/main/resources/application.yml

```diff
@@ -6,6 +6,10 @@ springdoc:
 spring:
   profiles:
     active: http
+  data:
+    web:
+      pageable:
+        one-indexed-parameters: true #Allow the UI to query page 1 instead of 0
```

Comment:
Does this mean that page 0 does not exist?

Reply:
If you try to call page 0 or page 1 through the controller, it answers with page 1.

Reply:
Okay, so there will be a difference between a consumer of the API (1 to n) and the logic in the backend with this parameter set to true?

Reply:
Yes, with this set to true, for the end-user and the UI, it corresponds to the usual way to count.
This allows to just display the property without having to pay more attention to it.